packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js

```diff
@@ -8,8 +8,11 @@ describe('tocco-ui', () => {
     describe('typeFormatters', () => {
       describe('PhoneFormatter ', () => {
         test('should format valid phone numbers', () => {
-          const wrapper = shallow(<PhoneFormatter value="+41443886000"/>)
-          expect(wrapper.text()).to.equal('+41 44 388 60 00')
+          const wrapper = mount(<PhoneFormatter value="+41443886000"/>)
+          return wrapper.instance().importLibPhoneNumber().then(() => {
```

Comment:
you could introduce a callback that gets invoked as soon as the lib is loaded and use it in the test. i would like that more than invoking a class method like this.